boring-sys/Cargo.toml

```diff
@@ -60,6 +60,9 @@ rustdoc-args = ["--cfg", "docsrs"]
 # Use a FIPS-validated version of boringssl.
 fips = []
 
+# Use a precompiled, FIPS validated boringssl.
+fips-precompiled = []
+
 # Link with precompiled FIPS-validated `bcm.o` module.
 fips-link-precompiled = []
```

Comment:
The descriptions for each of these flags all sound basically the same. I would be more specific about what the new flag actually does, in particular how it differs from "fips".

Also, "fips-precompiled" is very similar to "fips-link-precompiled".

Reply:
I added more detailed documentation in https://github.com/cloudflare/boring/pull/338/files#diff-902ee088de98c256522122e618b15e1fb66c1e57f7e30a0d108cefcb1af7df19R22 since it's the most popular entrypoint to the crate. I can copy documentation over to `boring-sys/Cargo.toml` if you'd like

Reply:
Copied the relevant documentation over